vignettes/quick-start.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "Quick Start"
+author: "R Validation Hub"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Quick Start}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+library(riskmetric)
+library(dplyr)
+library(tibble)
+
+options(repos = "https://cran.rstudio.com")
+
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
```

Comment:
Suggest to break the sentence below into multiple short sentence. 

I provide my version for your consideration. 

`riskmetric` provides a workflow to evaluate the quality of a set of R packages that involve four major steps. The workflow can help users to make a robust choice of high quality R package, improve package reliability and prove the validity of R packages in a regulated industry. The four steps include:

Reply:
Fixed paragraph in [73400c7](https://github.com/pharmaR/riskmetric/pull/29/commits/73400c7d8993544e04c957238b8a1cbd323a5129)